BLE Indoor Positioning/src/main/java/com/nexenio/bleindoorpositioning/location/distance/BeaconDistanceCalculator.java

```diff
@@ -23,6 +23,22 @@ public static float calculateDistanceTo(Beacon beacon) {
         return calculateDistanceTo(beacon, beacon.getFilteredRssi());
     }
 
+    /**
+     * Use Pythagoras with altitude to calculate more accurate distance to the beacon, if the
```

Comment:
Explain why someone would want to use this method instead of the normal `calculateDistanceTo (...)`